### DISCUSSION TITLE
Traductions

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#34] [Show and tell]
Dans le contexte de Laravel:
@claraquintela 
En gros, dans le input tu écris ça 

- pour anglais: name="ingredient_en" value="{{old('ingredient_en')}}"
- pour français: name="ingredient_fr" value="{{old('ingredient_fr')}}"

                    <form action="{{ route('ingredient.store') }}" method="POST">
                        @csrf
                        <div class="mb-3">
                            <label for="ingredient_en" class="form-label">Ingredient in English</label>
                            <input type="text" class="block appearance-none w-full py-1 px-2 mb-1 text-base leading-normal bg-white text-gray-800 border border-gray-200 rounded" id="ingredient_en" name="ingredient_en" value="{{old('ingredient_en')}}">
                        </div>
                        <div class="mb-3">
                            <label for="ingredient_fr" class="form-label">Ingredient in French</label>
                            <input type="text" class="block appearance-none w-full py-1 px-2 mb-1 text-base leading-normal bg-white text-gray-800 border border-gray-200 rounded" id="ingredient_fr" name="ingredient_fr" value="{{old('ingredient_fr')}}">
                        </div>
                        <button type="submit" class="inline-block align-middle text-center select-none border font-normal whitespace-no-wrap rounded py-1 px-3 leading-normal no-underline bg-blue-600 text-white hover:bg-blue-600">Save</button>
                    </form>

**Comment by claraquintela** [comment]
Merci :)
Ça m'aidera bcp! :)

Em qua., 26 de jun. de 2024 às 11:12, René de Montigny <
***@***.***> escreveu:

**Comment by claraquintela** [comment]
Salut, René! @CollRen 

J'ai besoin de changer le type de donnés de la column "type", dans la table Privileges pour JSON, mais je n'arrive pas. Est-ce que tu a mis une protection lors de la creation automatique de la db? 

Merci de me dire. J'essaye de faire rentré un objet json pour les traductions dinamiques.

**Reply by CollRen** [reply]
Non tu devrais pouvoir le faire...

**Comment by claraquintela** [comment]
Alter ne fonctionne pas.
J’ai essaye aussi effacer la columne et la récrier comme json. Pourtant ele
s’enregistre toujours comme longtext. J’ai le faire deux fois
